CSS/Styles.css

```diff
@@ -0,0 +1,96 @@
+* {
+  width: 320px;
+  margin: 0;
+  padding: 0;
+  font-family: 'Playfair Display', serif;
+  font-family: 'Poppins', sans-serif;
+}
+
+body {
+  background-color: #e5e5e5;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.menu {
+  position: absolute;
+  display: flex;
+  width: 20px;
+  height: 14px;
+  left: 284px;
+  top: 19px;
+}
+
+.logo {
+  display: flex;
+  width: 111px;
+  height: 74px;
+  left: 0;
+  top: 0;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.deco {
+  padding: 32px 0;
+  display: flex;
+  width: 320px;
+  height: 160px;
+  left: 0;
+  top: 472px;
+}
+
+p {
```

Comment:
[OPTIONAL] I recommend against the use of tag names in your CSS file as this will cause issues when you develop your portfolio further and have multiple tags of the same type that you may want to style differently. I suggest you get in the habit of adding classes to all of your tags from early on. This comment applies to all the other tags you are using in your CSS code, I'm marking just this one for brevity.